Refit/FormValueDictionary.cs

```diff
@@ -3,16 +3,23 @@
 using System.Reflection;
 using System.Collections.Generic;
 using System.Linq;
-using System.Threading.Tasks;
 using Newtonsoft.Json;
 
 namespace Refit
 {
-    class FormValueDictionary : Dictionary<string, string>
+    /// <summary>
+    /// Transforms a form source from a .NET representation to the appropriate HTTP form encoded representation.
+    /// </summary>
+    /// <remarks>Performs field renaming and value formatting as specified in <see cref="QueryAttribute"/>s and
+    /// <see cref="RefitSettings.FormUrlEncodedParameterFormatter"/>. Note that this is not a true dictionary, rather, a
+    /// form of MultiMap. A given key may appear multiple times with the same or different values.</remarks>
+    class FormValueDictionary : IEnumerable<KeyValuePair<string, string>>
```

Comment:
Is having this be called a `Dictionary` confusing? Should I rename it? Outside of `FormValueDictionaryTests` there is only 1 line that would be affected by a rename.

Reply:
It can be renamed as it's an internal class